up-l1/cloudevents.adoc

```diff
@@ -74,48 +72,48 @@ This mapping MUST be used by link:../upclient.adoc[uProtocol Client Libraries] w
 |String
 a|CloudEvent specification version. This is *NOT* the uProtocol version
 
-* MUST be set to `1.0`
+* *MUST* be set to `1.0`
 
 
 |`type`
 |https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md#type[`type`]
 |String
 a|The type of the message
 
-* MUST be set to the value of the `uprotocol.ce_name` option defined for the
-link:../up-core-api/uprotocol/uattributes.proto[UMessageType enum].
+* *MUST* be set to the value of the `uprotocol.ce_name` option defined for the
+xref:../up-core-api/uprotocol/uattributes.proto[UMessageType enum].
```

Comment:
Since this is a reference to a protofile you should stay with link and not xref right?

Reply:
absolutely